src/components/avatar-editor/README.md

```diff
@@ -9,6 +9,8 @@ This is our avatar editor component. You can use it to change your profiles', or
 | src | string | The url of the src attribute on the image tag. | `none` |
 | onChange | function | It's responsible for creating a valid url for the img tag by converting a blob to a valid url string. | `none` |
 | isProfileAvatar | boolean | `isProfileAvatar` determines whether the component should render an UserAvatar or an org logo as its child. | `true` |
+| buttonLabelAvatar | node | You can specify the label of the Button, when isProfileAvatar is `false`. | `'Upload picture'` |
```

Comment:
buttonLabelAvatar and buttonLabelOrg should be removed and please, add the following props:

```
buttonLabel: PropTypes.node,
borderRadius: PropTypes.number (default: 100)
```

Reply:
I would like to put the `borderRadius` prop in, but there is a problem:
- If we eliminate `isProfileAvatar` prop and use only `borderRadius`, then we would lose the two separate placeholder figures if `AvatarEditor` is empty.

Reply:
I'm not sure if I understand it well, but what I'm 100% percent sure that I don't want this component to know about anything whether it contains a profile picture or an organization logo.

On the other end, if we want to use it on the account page, we would set the border-radius to '100%' or `50%` to get a circle. In case of the organization page we would set the border-radius to `20`:

```
// account
<AvatarEditor
  borderRadius="100%"
/>

// org
<AvatarEditor
  borderRadius={ 20 }
/>
```

If you want the placeholder to be dynamic, you have to provide a solution to set it by the end user. It means we have to have a prop called `placeholder`. The default can be the user shape.